packages/backend/src/apps/frappe/actions/create-doc.js

```diff
@@ -0,0 +1,34 @@
+import defineAction from '../../../helpers/define-action.js';
+import { getFrappeDocumentAPIUrl } from '../common/utils.js';
+
+export default defineAction({
+    name: 'Create New Document',
+    key: 'createDoc',
+    description: 'Create a new document in Frappe.',
+    arguments: [
+        {
+            label: 'Document Type',
+            key: 'doctype',
+            type: 'string',
+            required: true,
+            description: 'The type of the doctype to retrieve.',
+            variables: true,
```

Comment:
If you do not get the data dynamically (eg. list of doc types) please use `variables: false`.
Applies to all the actions.